src/libs/fades.liq

```diff
@@ -49,38 +49,43 @@ def mkfade(
   end
 
   shape =
-    if type == "sin" then
+    if
```

Comment:
It would be nice to apply auto-indentation on main, so that we can see the reall diff here.

Reply:
Yeah, this is the `v2.2.x` branch, the auto-indentation wasn't applied yet. You can see a proper diff in the `main` backport: https://github.com/savonet/liquidsoap/pull/3704